src/ncp/ncp_base.hpp

```diff
@@ -513,6 +530,9 @@ class NcpBase
     bool mRequireJoinExistingNetwork;
     bool mIsRawStreamEnabled;
 
+    bool mIsBoundToRadio;
```

Comment:
Should we change this to some kind of state variable, something like `{ Unbound, RadioBound, ThreadBound }` and prevent calls to ot* if we are `RadioBound` and prevent calls to otPlatRadio* if we are `ThreadBound`?